### DISCUSSION TITLE
a couple of typos and questions

### DISCUSSION THREAD

**Discussion by yafm** [hickeng/financial#114] [General]
Pretty fancy and a lot of hard work! Thanks.

I tried the "binary" file in a desktop excel 2007 (not excel 365). and it didn't like it. I don't believe it works there, FYI

I am using the google sheets version and see the following typos:

- ESPP tab: Purchase Date 2019-02-28 grant date market value should be 153.26 not 123.66
- ESPP tab: Purchase Date 2019-08-31 grant date market value should be not 153.26 not 178.2

Once entered, is this sheet supposed to calculate cost basis? I am not clear what results I should check to see it did some calculations. I only see the gain part being calculated.

On RSU tab: Should the user enter the pre-12/28/2018 numbers (when the dividend payout and shares/prices were adjusted) or post that event's numbers? 

And similar question for RSU as I asked for ESPP: what is supposed to happen upon entry? Is it going to calculate cost basis or something?

**Comment by hickeng** [comment]
> I tried the "binary" file in a desktop excel 2007 (not excel 365). and it didn't like it. I don't believe it works there, FYI
> 
Thanks for letting me know. O365 if free for use online and you can open there to get a populated copy for a CPA.

> I am using the google sheets version and see the following typos:
> 
> * ESPP tab: Purchase Date 2019-02-28 grant date market value should be 153.26 not 123.66
> * ESPP tab: Purchase Date 2019-08-31 grant date market value should be not 153.26 not 178.2
> 

These aren't typos but, I think, the result of you entering the purchase periods at a different time. I'm surprised you have two - most people with this only have one skewed value when they first joined the ESPP program.

If you update the pre-populated values to match yours, the sheet will use your values.

> Once entered, is this sheet supposed to calculate cost basis? I am not clear what results I should check to see it did some calculations. I only see the gain part being calculated.

The sheet calculates the basis you need for reporting post-merger sales, and the Form 8949 values you need for reporting the merger.
If you want to see the basis entering the merger (this would only be interesting if you sold pre-merger) or any of the other intermediate values, expand the hidden columns. This is true for both RSU and ESPP.

> On RSU tab: Should the user enter the pre-12/28/2018 numbers (when the dividend payout and shares/prices were adjusted) or post that event's numbers?

You're entering the values from the Purchase Confirmation (ESPP) or Release Confirmation (RSUs). The sheet does the adjustment for the two return of capital events based on the date you enter for receiving the shares.

I've got #112 for adding to the ESPP doc about how to add/edit rows. I'll roll the "why I've got different value" explanation in to that.
